crates/router/src/connector/bankofamerica/transformers.rs

```diff
@@ -722,6 +745,44 @@ pub struct ClientReferenceInformation {
 pub struct ClientProcessorInformation {
     avs: Option<Avs>,
     card_verification: Option<CardVerification>,
+    processor: Option<ProcessorResponse>,
+    network_transaction_id: Option<String>,
+    approval_code: Option<String>,
+    merchant_advice: Option<MerchantAdvice>,
+    response_code: Option<String>,
+    ach_verification: Option<AchVerification>,
+    system_trace_audit_number: Option<String>,
```

Comment:
Can you please confirm none of the structs contain sensitive data, if so can you please mask it as a secret